fakeData.js

```diff
@@ -0,0 +1,46 @@
+var faker = require('faker');
+fakeData = [];
+
+function getRandomNum(min, max) {
+    return Math.floor(Math.random() * (max - min) + min);
+}
+
+for (let listingId = 0; listingId < 200; listingId++){
+    listingQuestionsAndAnswers = {}
+
+    listingQuestionsAndAnswers.id = listingId;
+    listingQuestionsAndAnswers.questions = [];
+
+    for (let i = 0; i < 2; i++){
```

Comment:
I would consider randomizing the number of questions per listing and answers per question. That way you have that variation in data available when you work on the front end.